packages/coreutils/src/settingregistry.ts

```diff
@@ -880,11 +882,14 @@ export namespace SettingRegistry {
 
     // If a user shortcut collides with another user shortcut warn and filter.
     user = user.filter(shortcut => {
-      const keys = shortcut.keys.join(RECORD_SEPARATOR);
+      const keys = normalizeKeys(shortcut).join(RECORD_SEPARATOR);
       const { selector } = shortcut;
 
       if (!keys) {
-        console.warn('Shortcut skipped because `keys` are [""].', shortcut);
+        console.warn(
+          'Shortcut skipped because `keys` and platform keys are [""].',
```

Comment:
How about "Shortcut skipped because no shortcut keys were applicable on this platform"